### DISCUSSION TITLE
Future of this? Microsoft announced discontinuation of WSA

### DISCUSSION THREAD

**Discussion by xshunin** [LSPosed/MagiskOnWSALocal#787] [General]
So what will happen to this?
Here's the news: https://learn.microsoft.com/en-us/windows/android/wsa/
This was the cleanest way to use Google Services on Windows :(

**Comment by Philaupatte** [comment]
Hi,
Yes indeed, this is going to be closed and yes it was a very clear and clean solution, so I am looking for a replacement solution.

I try Bluestack and it works but I feel like using a all long train to cary out only a feather.

Have you any suggestion to recommend the best replacement solution?

**Comment by xshunin** [comment]
Unfortunetly I don't. I liked WSA best because it was the purest solution there is. Every other pre-made emulator comes with ads. I haven't even found one that you can buy and get it ad free (I think there was one a few years ago?)

**Comment by thelittleerik** [comment]
I seriously can't believe WSA will close up shop. WIll there atleast be mirrors for WSA so in the future I can install it regardless of support? It's a seriously helpful tool.
